nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -514,32 +527,30 @@ public boolean equals(Object o) {
     if (this == o) {
       return true;
     }
-    if (!(o instanceof AccessPath)) {
+    if (o == null || getClass() != o.getClass()) {
```

Comment:
Any significant differences here between using `instanceof` vs comparing `.getClass()`? I assume no support for subtyping, but `AccessPath` is `final`, so it should be ok?